pkg/apis/config/v1alpha2/meshrootcertificate.go

```diff
@@ -21,6 +21,10 @@ type MeshRootCertificate struct {
 	// +optional
 	Spec MeshRootCertificateSpec `json:"spec,omitempty"`
 
+	// Intent of the MeshRootCertificate resource
+	// +optional
+	Intent MeshRootCertificateIntent `json:"intent,omitempty"`
```

Comment:
Could you also update the `preset-mesh-root-certificate.yaml` to include the `intent` field? Thanks!